src/utils/indexedDB.ts

```diff
@@ -0,0 +1,62 @@
+const DB_NAME = 'exampleDB';
+const DB_VERSION = 1;
+const STORE_NAME = 'certificates';
+
+interface Certificate {
+  id?: number;
+  supplier: string;
+  certificateType: string;
+  validFrom: string;
```

Comment:
validFrom and validTo are fields to mark validity of Certificate, so they should be dates. Please fix it and also display them in correct format in table, e.g. '22.07.2024'

Reply:
> validFrom and validTo are fields to mark validity of Certificate, so they should be dates. Please fix it and also display them in correct format in table, e.g. '22.07.2024'

Here implemented the id to be auto increments , so the optional has to be there, and i will no longer need to add certificate id.